src/whisper_at_train/run_combined_training.sh

```
@@ -33,53 +43,89 @@ wa_start=36
+  --lr_adapt "${lr_adapt}" \
+  --lr_patience "${lr_patience}" \
   --num-workers 8 \
-  --pretrained_model ${pretrained_model} \
-  --weight_decay ${weight_decay}
-  #--freeze_original_classes
\ No newline at end of file
+  --pretrained_model "${pretrained_model}" \
+  --weight_decay "${weight_decay}"
+
+# 훈련 완료 상태 확인
+if [[ $? -eq 0 ]]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)